pkg/data/attributes.go

```diff
@@ -42,6 +42,7 @@ type Block struct {
 	PhysicalBlockSize *string `json:"physicalBlockSize,omitempty" yaml:"physicalBlockSize,omitempty"`
 	Vendor            *string `json:"vendor,omitempty" yaml:"vendor,omitempty"`
 	Model             *string `json:"model,omitempty" yaml:"model,omitempty"`
+	Wwn               *string `json:"wwn,omitempty" yaml:"wwn,omitempty"`
```

Comment:
Mind using a consistent case (uppercase) for this across the PR? [reference](https://go.dev/wiki/CodeReviewComments#initialisms)

`Wwn` -> `WWN`